http/http-minimum-reactive/src/test/java/io/quarkus/ts/http/minimum/reactive/OperatorResourceIT.java

```diff
@@ -0,0 +1,26 @@
+
+package io.quarkus.ts.http.minimum.reactive;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.http.ContentType;
+
+@QuarkusIntegrationTest
+public class OperatorResourceIT {
+
+    @Test
```

Comment:
Consider disabling it for 3.15.2 and lesser, because `3.15.3` is not yet released. I don't know when this is scheduled for a backport.

Reply:
Fix is in 3.15 branch, merged last week in backports PR nr. 2